.travis.yml

```diff
@@ -7,7 +7,7 @@ go:
 install: true
 
 before_install:
-  go get github.com/codegangsta/cli && go get github.com/stretchr/testify/assert
+  bin/deps
```

Comment:
Would `go get -t ./...` suffice here? It would fetch all dependencies for each package.